src/utils/checkout.js

```diff
@@ -189,3 +190,58 @@ export const POST_OFFICE_NUMBER = 'Відділення № ';
 
 export const getCurrentCurrency = (currency, language = 1) =>
   currency === DEFAULT_CURRENCY ? CHECKOUT_TITLES[language].UAH : CHECKOUT_TITLES[language].USD;
+
+export const setUserValues = (values, userData, deliveryType) => {
+  const { firstName, lastName, email, phoneNumber } = userData;
+  const result = { ...values, firstName, lastName, email, phoneNumber };
+  if (
+    (deliveryType === deliveryTypes.NOVAPOSTCOURIER ||
+      deliveryType === deliveryTypes.UKRPOSTCOURIER) &&
+    userData.address
+  ) {
+    const { city, street, buildingNumber: house, appartment: flat } = userData.address;
+    return {
+      ...result,
+      city,
+      street,
+      house,
+      flat
+    };
+  }
+  return result;
+};
+
+const COURIER = 'COURIER';
```

Comment:
Is it ok if we have magic strings within util files? Even if it's not regulated strictly, there is still **consts/** folder, so maybe it's better to move this const there.